process/mock/runTypeComponentsStub.go

```diff
@@ -48,15 +51,16 @@ func NewSovereignRunTypeComponentsStub() *RunTypeComponentsStub {
 	requestHandlerFactory, _ := requestHandlers.NewSovereignResolverRequestHandlerFactory(rt.RequestHandlerFactory)
 
 	return &RunTypeComponentsStub{
-		AdditionalStorageServiceFactory:        &testscommon.AdditionalStorageServiceFactoryMock{},
-		ShardCoordinatorFactory:                sharding.NewSovereignShardCoordinatorFactory(),
-		NodesCoordinatorWithRaterFactory:       &testscommon.NodesCoordinatorFactoryMock{},
-		RequestHandlerFactory:                  requestHandlerFactory,
-		AccountCreator:                         &stateMock.AccountsFactoryStub{},
-		OutGoingOperationsPool:                 &sovereignMocks.OutGoingOperationsPoolMock{},
-		DataCodec:                              &sovereignMocks.DataCodecMock{},
-		TopicsChecker:                          &sovereignMocks.TopicsCheckerMock{},
-		RequestersContainerFactoryCreatorField: requesterscontainer.NewSovereignShardRequestersContainerFactoryCreator(),
+		AdditionalStorageServiceFactory:            &testscommon.AdditionalStorageServiceFactoryMock{},
+		ShardCoordinatorFactory:                    sharding.NewSovereignShardCoordinatorFactory(),
+		NodesCoordinatorWithRaterFactory:           &testscommon.NodesCoordinatorFactoryMock{},
+		RequestHandlerFactory:                      requestHandlerFactory,
+		AccountCreator:                             &stateMock.AccountsFactoryStub{},
+		OutGoingOperationsPool:                     &sovereignMocks.OutGoingOperationsPoolMock{},
+		DataCodec:                                  &sovereignMocks.DataCodecMock{},
+		TopicsChecker:                              &sovereignMocks.TopicsCheckerMock{},
+		RequestersContainerFactoryCreatorField:     requesterscontainer.NewSovereignShardRequestersContainerFactoryCreator(),
+		ValidatorAccountsSyncerFactoryHandlerField: syncerFactory.NewSovereignValidatorAccountsSyncerFactory(),
```

Comment:
Maybe use mock here ValidatorAccountsSyncerFactoryMock?

Reply:
This "stub" is not a stub actually, it uses mostly real comps;
Also, got the same "cyclic" import issue here, there is a todo to fix it